cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,393 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/briandowns/spinner"
+	goyaml "github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	"log"
+	"math"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const MAXTARGETSHOW = 10
+const POD = "pod"
```

Comment:
Rather than re-defining this, its better to import `chaostypes.DisruptionLevelPod` and use that for comparison. Same with `"node"` which is used at L34